docker/docker-compose.yml

```diff
@@ -98,6 +98,9 @@ services:
       BLOCKCHAIN_PORT: 8546
       HASH_TYPE: poseidon
       LOG_LEVEL: debug
+      # right now it doesn't matter what is IS_CHALLENGER flag's value is because
```

Comment:
Then `IS_CHALLENGER` is not really used. It is not clear if this flag is needed anymore